src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,58 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String COMA = ",";
+    private static final String RESULT = "result";
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final int INDEX_OF_AMOUNT = 1;
+    private static final int INDEX_OF_OPERATION = 0;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String data = getDataFromFile(fromFileName);
+        writeToFile(toFileName, data);
+    }
+
+    private String getDataFromFile(String fromFileName) {
+        int countOfSupply = 0;
+        int countOfBuy = 0;
```

Comment:
```suggestion
        int supplyAmount = 0;
        int buyAmount = 0;
```
count usually used for counters, like in loops

Reply:
renamed